modP2pImpl/src/org/aion/p2p/impl1/P2pMgr.java

```diff
@@ -176,7 +178,8 @@ public void run() {
             tcpServer = ServerSocketChannel.open();
             tcpServer.configureBlocking(false);
             tcpServer.socket().setReuseAddress(true);
-            tcpServer.socket().bind(new InetSocketAddress(Node.ipBytesToStr(selfIp), selfPort));
+            tcpServer.socket().setReceiveBufferSize(128*1024);
```

Comment:
a comment explaining the number would be helpful

Reply:
updated